packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -2,13 +2,15 @@
 <!--BEGIN STABILITY BANNER-->
 ---
 
-![cfn-resources: Stable](https://img.shields.io/badge/cfn--resources-stable-success.svg?style=for-the-badge)
+| Features | Stability |
+| --- | --- |
+| CFN Resources | Stable |
+| Higher level constructs for User Pools | Developer Preview |
+| Higher level constructs for Identity Pools | Not Implemented |
 
 > All classes with the `Cfn` prefix in this module ([CFN Resources](https://docs.aws.amazon.com/cdk/latest/guide/constructs.html#constructs_lib)) are always stable and safe to use.
 
-![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
-
-> The APIs of higher level constructs in this module are experimental and under active development. They are subject to non-backward compatible changes or removal in any future version. These are not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be announced in the release notes. This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+> Higher level constructs in this module that are marked Experimental or Developer Preview are under active development. They are subject to non-backward compatible changes or removal in any future version. These are not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be announced in the release notes. This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
```

Comment:
Don’t we want some info that developer preview is not expected to change dramatically? Isn’t the whole point to signal to users “you can trust that this is likely to be the final API?” Otherwise why do we need this distinction?

Reply:
Yes, fair. I've added.